bsc-plugin/src/lib/rooibos/RooibosSession.ts

```diff
@@ -89,7 +88,12 @@ export class RooibosSession {
             }
         }
         if (mainFunction) {
-            editor.addToArray(mainFunction.func.body.statements, 0, new RawCodeStatement(`Rooibos_init("${this.config?.testSceneName ?? 'RooibosScene'}")`));
+            const initCall = mainFunction.func.body.findChild(f => isCallExpression(f) && isVariableExpression(f.callee) && f.callee.name.text.toLowerCase() === 'rooibos_init', {
+                walkMode: WalkMode.visitAllRecursive
+            });
+            if (!initCall) {
+                editor.addToArray(mainFunction.func.body.statements, 0, new RawCodeStatement(`Rooibos_init("${this.config?.testSceneName ?? 'RooibosScene'}")`));
```

Comment:
I've considered changing this line to use codegen methods instead of `RawCodeStatement`, but decided not to in order to keep the PR small and concise. However, it's something that could be considered in a future refactor if we want more safety.

```suggestion
                editor.addToArray(mainFunction.func.body.statements, 0, new ExpressionStatement(createCall(
                    createVariableExpression('rooibos_init'),
                    [
                        createVariableExpression(`"${this.config?.testSceneName ?? 'RooibosScene'}"`)
                    ]
                )));
```

Reply:
It's probably not a bad idea. @TwitchBronBron and I where looking today to move the `modifyAssertions` to proper ast transformations.

Reply:
agree; good idea, we can revisit in another pr.